documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -47,3 +47,51 @@ The precopy stage runs until the cutover stage is started manually or is schedul
 The VMs are shut down during the cutover stage and the remaining data is migrated. Data stored in RAM is not migrated.
 
 You can start the cutover stage manually by using the {project-short} console or you can schedule a cutover time in the `Migration` manifest.
+
+[id="warm-migration-versus-cold-migration"_{context}]
+== Cold migration vs. warm migration: Which is right for your migration?
+
+Both warm migration and cold migration have advantages and limitations. In general, warm migrations can take longer than cold migrations, but warm migrations involve less downtime than cold migrations.
+
+The table that follows describes the advantages and limitations of each type of migration. It assumes that you have installed Red Hat Enterprise Linux (RHEL) 9 on the {ocp} platform on which you installed {project-short}.
+
+[cols="1,1,1",options="header"]
+.Advantages and limitations of cold and warm migrations
+
+|===
+| |Cold migration |Warm migration
+
+|*Duration*
+| Each block is copied only once, making the duration of the migration shorter.
+| A block might be copied in more than one snapshot. {project-short} ensures that only the most recent snapshot is used, but eliminating duplicate snapshots adds time to the duration of the migration.
+
+|*Fail fast*
+| Each block is converted to be compatible with {ocp-short} and, if the conversion is successful, the block is transferred. If a block cannot be converted, the migration fails immediately.
+| All blocks are transferred before any are converted. This means that it takes longer for a warm migration to fail than a cold migration, because a cold migration fails before all blocks are transferred.
+
+|*Tools*
+| RHEL 9 and `virt-v2v` in all cases.
```

Comment:
RHEL 9 and virt-v2v are internal details, thus what about simply saying "MTV only"